### PR TITLE
Linda's attempt (issue #16)

### PR DESCRIPTION
**Pull request by intoCareers**
Another try

### CODE REVIEW

**Comment by llhowe** [comment]
@intoCareers please request someone to review this pull  request--not Chris:)

**Review by cwc-products — DISMISSED** [review]
Please link the comments with the original ticket for this PR?

**Comment by intoCareers** [comment]
How do you link to issue?

**Comment by intoCareers** [comment]
Please someone review :-)

**Comment by intoCareers** [comment]
This is to solve #16

**Review by papajeremy — APPROVED** [review]
This looks good.

**Review by delraydavis — APPROVED** [review]
looks right!

File: app.js [hunk 1]
```diff
@@ -1,2 +1,3 @@
 const { goDucks } = require('./go-ducks');
-console.log(goDucks.quack());
\ No newline at end of file
+goDucks.appendMessages([() => 'IT\'S']);
```
Comment (cwc-products) [inline app.js:3]: This looks good!

**Review by cwc-products — APPROVED** [review]

**Comment by cwc-products** [comment]
@intoCareers please resolve the conflicts.

**Review by cwc-products — CHANGES_REQUESTED** [review]
Please resolve conflicts.

**Comment by intoCareers** [comment]
Resolved -- what now?

**Review by cwc-products — APPROVED** [review]

**Comment by cwc-products** [comment]
@intoCareers changes are good to go... unless someone beats you... but you already merged so I guess you win.

**Pull request merged**